src/scss/components/optional/_cuisines-widget.scss

```diff
@@ -0,0 +1,127 @@
+/**
+* Components > Cuisines Widget
+* =================================
+*
+* Example: Homepage – Popular cuisines.
+*
+* The `c-cuisinesWidget` component is an optional mixin within Fozzie.
+* If you'd like to use it in your project you can include it by adding `@include cuisinesWidget();` into your SCSS dependencies file.
+*
+* Documentation:
+* https://fozzie.just-eat.com/styleguide/ui-components/cuisines-widget
+*/
+
+$cuisinesWidget-titleColour: $white;
+$cuisinesWidget-defaultBackground: $brand--red;
+$cuisinesWidgetGrid-breakpoint-narrow: 570px;
+$cuisinesWidgetGrid-breakpoint-wide: 1241px;
+$cuisinesWidget-boxShadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
+$cuisinesWidget-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0 rgba(0, 0, 0, 0.04);
+$cuisinesWidget-gradient: linear-gradient(to top, rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0));
+
+@mixin cuisinesWidget() {
+
+   .c-cuisinesWidget {
+       display: block;
+       border-radius: 8px;
+       overflow: hidden;
+       flex: 0 0 auto;
+       position: relative;
+       background-color: $cuisinesWidget-defaultBackground;
+       box-shadow: $cuisinesWidget-boxShadow;
+
+       &:hover {
+           box-shadow: $cuisinesWidget-hoverBoxShadow;
+       }
+
+       .c-cuisinesWidget-name {
+           position: absolute;
+           bottom: 0;
+           left: 0;
+           width: 100%;
+           padding: spacing(x2);
+           color: $cuisinesWidget-titleColour;
+           text-align: center;
+           background-image: $cuisinesWidget-gradient;
+
+           @media screen and (min-width: $cuisinesWidgetGrid-breakpoint-wide) {
```

Comment:
Should use the `media-content` part of @include-media to [extend the breakpoint context](https://include-media.com/documentation/#mixin-media-context) for this component and add the new breakpoints in.

Reply:
✅